assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -432,9 +432,9 @@ export const MESSAGES = {
           </li>
           <li>
             {sprintf(
-              /* translators: %d: maximum percentage of characters, depending on number of pages. */
+              /* translators: 1: maximum percentage of characters, depending on number of pages. */
               __(
-                'Keep the total number of text-heavy pages under %d% of total number of story pages',
+                'Keep the total number of text-heavy pages under %1$d%% of total number of story pages',
```

Comment:
If there's only one placeholder, just using `%d` instead of numbered placeholders (`%1$d`) is preferred.

Still need the dobule `%%` to get a single percentage sign of course. Alternatively do `MAX_CHARACTER_PERCENTAGE + '%'` below